packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-logs/job-logs.html

```diff
@@ -0,0 +1,40 @@
+<div data-ui-view>
+    <div class="logger col-md-12" data-scroll-glue>
```

Comment:
```suggestion
    <div class="logger" data-scroll-glue>
```
Not needed therefore :) (Also to use a `col-md` the parent need to have a `.row`